_pages/rfc-pairing.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: pageminimal
+title: "RFC [r] - WeAllJS #pairing"
+permalink: /rfc-pairing
+---
+
+# RFC [r] - WeAllJS #pairing
+
+[ratified 2016-09-17]
+{: text-italic}
+
+Learning is cool! Programming is hard! Learning with someone else is often way more productive than trying to learn alone.
+We All JS  strives to be a community where people learning something new, no matter their level of skill can find support.
```

Comment:
This might read better as: "We All JS  strives to be a community where people, no matter their level of skill can find support learning something new."